sdk/provisioning/Azure.Provisioning.ServiceBus/tests/BasicServiceBusTests.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.ServiceBus.Tests;
+
+public class BasicServiceBusTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    [Description("https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.servicebus/servicebus-create-queue/main.bicep")]
+    public async Task CreateServiceBusQueue()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(
+            ctx =>
+            {
+                BicepParameter location = BicepParameter.Create<string>(nameof(location), BicepFunction.GetResourceGroup().Location);
+                location.Description = "The SB location.";
+
+                BicepParameter queueName = BicepParameter.Create<string>(nameof(queueName), "orders");
+                queueName.Description = "The name of the SB queue.";
+
+                ServiceBusNamespace sb =
+                    new(nameof(sb), ServiceBusNamespace.ResourceVersions.V2021_11_01)
+                    {
+                        Location = location,
+                        Sku = new ServiceBusSku { Name = ServiceBusSkuName.Standard },
+                    };
+
+                ServiceBusQueue queue =
+                    new(nameof(queue), ServiceBusNamespace.ResourceVersions.V2021_11_01)
+                    {
+                        Parent = sb,
+                        Name = queueName,
+                        // Hack around TimeSpan not serializing ISO durations
+                        // correctly using a Bicep string literal expression.
+                        // TODO: Change these to regular strings when patched.
+                        LockDuration = new StringLiteral("PT5M"),
+                        MaxSizeInMegabytes = 1024,
+                        RequiresDuplicateDetection = false,
+                        RequiresSession = false,
+                        DefaultMessageTimeToLive = new StringLiteral("P10675199DT2H48M5.4775807S"),
+                        DeadLetteringOnMessageExpiration = false,
+                        DuplicateDetectionHistoryTimeWindow = new StringLiteral("PT10M"),
+                        MaxDeliveryCount = 10,
+                        AutoDeleteOnIdle = new StringLiteral("P10675199DT2H48M5.4775807S"),
```

Comment:
Or you can deal with services who use ISO8601 durations even though no one in the world supports them out of the box and our `TimeSpan` will serialize incorrectly.   Expressions have you covered.